src/main/webapp/index.html

```diff
@@ -166,13 +166,35 @@ <h2 class="panel-row__header u-align-with-first-panel">Build your plan:</h2>
     <div class="panel-row__content">
       <div class="panel">Assign</div>
       <div class="panel">Plan</div>
-      <div class="panel">Go
-        <div id="go"></div>
+      <div class="panel">
```

Comment:
Can you create a utility class `.u-border-go` and change the border of this panel to match some unique color (I'm thinking a material green for this panel - currently saved as a variable actually `--color-green`). Any color is good, but I think the black looks too plain for our colour palette.

Reply:
Got it - changed to orange. Thanks!